test/e2e/tests/account/incremental-security.spec.ts

```diff
@@ -0,0 +1,104 @@
+import { withFixtures, WALLET_PASSWORD } from '../../helpers';
+import FixtureBuilder from '../../fixture-builder';
+import { Driver } from '../../webdriver/driver';
+import { Suite } from 'mocha';
+import HomePage from '../../page-objects/pages/home/homepage';
+import OnboardingCompletePage from '../../page-objects/pages/onboarding/onboarding-complete-page';
+import OnboardingMetricsPage from '../../page-objects/pages/onboarding/onboarding-metrics-page';
+import OnboardingPasswordPage from '../../page-objects/pages/onboarding/onboarding-password-page';
+import SecureWalletPage from '../../page-objects/pages/onboarding/secure-wallet-page';
+import StartOnboardingPage from '../../page-objects/pages/onboarding/start-onboarding-page';
+import TestDapp from '../../page-objects/pages/test-dapp';
+
+/**
+ * Sends ETH to the current account from the dapp send-eth-with-private-key-test
+ * This dapp is only used in this test spec, so we keep the function here
+ * TODO: migrate this function to follow page object modal pattern when this dapp is used in other tests
+ */
+async function sendEthWithPrivateKey(driver: Driver) {
+  const sendEthWithPrivateKeyButton = '#send';
+  await driver.waitForSelector(sendEthWithPrivateKeyButton);
+  await driver.pasteFromClipboardIntoField('#address');
+  await driver.clickElement(sendEthWithPrivateKeyButton);
+  await driver.waitForSelector(
+    { css: '#success', text: 'Success' },
+    { timeout: 15000 },
+  );
+}
```

Comment:
mmm I would rather create a new class and add all these actions inside that class already in this PR. The reason for that is because if there's a new spec that needs to use this, we are not 100% sure a new class will be created, and given we are already migrating to POM in this PR, it might be safer to just do it here?

This might also be less confusing than now, as it seems we are opening the "regular" test dapp, but this is not the case for this spec.

What do you think?